content/en/blog/2022/collector-builder-sample/index.md

```diff
@@ -0,0 +1,314 @@
+accelerates compiling stripped-down Collector images with a simple YAML config
+file. This file declares which components to include in the build, and
+collector-builder includes only those components (and nothing else). This is in
+contrast to a default Collector build, where many more components are included
+in the compiled binary (even if they are not being enabled in the Collector’s
+runtime configuration). This difference is how you can achieve improvements over
+public images that include many excess components, with extra dependencies
+adding weight and security burden. Now, those extra components can’t take up
+space because they aren’t even available in the custom build.
+
+To show how this works we start with a sample config file in this repository
+which has been pre-filled with a few of the most relevant OpenTelemetry
+components for GCP. However, we encourage you to modify that sample file to fit
+your use case.
+
+For example, the following build file includes only the OTLP receiver and GCP
+exporter, along with a logging exporter:
+
+```
+receivers:
+  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
+    gomod: go.opentelemetry.io/collector v0.57.2
+
+exporters:
+  - import: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter
+    gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.57.2
+  - import: go.opentelemetry.io/collector/exporter/loggingexporter
+    gomod: go.opentelemetry.io/collector v0.57.2
+```
+
+Edit
+[this file](https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample/blob/main/build/local/builder-config.yaml)
+in the repository and run `make build` to automatically generate a local binary,
+or `make docker-build` to compile a container image.
+
+## Get up and running quickly on GKE
+
+For convenience, this repository includes the minimum Kubernetes manifests used
+to deploy the Collector in a GKE cluster, with a compatible
+[runtime configuration](https://github.com/GoogleCloudPlatform/opentelemetry-collector-builder-sample/blob/main/deploy/gke/simple/otel-config.yaml)
+for the sample collector-builder components built by default. When used
+together, the Make commands provided to build and push an image to Artifact
+Registry will automatically update those manifests in the repository to use the
+newly-created image, offering an end-to-end build and deployment reference.
+
+### Deploying the Collector in GKE
+
+As mentioned earlier in this post, the collector offers vendor-agnostic routing
+and processing for logs, metrics, and tracing telemetry data. For example, while
+you may already be using a collection agent on GKE (such as for metrics and
+logs), the collector can provide a pathway for exporting traces. Those traces
+can be sent to a GCP service like Cloud Trace, or an external provider for
```

Comment:
Please remove the comment "GCP service like Cloud Trace", e.g. "Those traces can be sent to an observability backend of your choice"